evap/urls.py

```diff
@@ -13,6 +14,8 @@
 
     path("logout", django.contrib.auth.views.LogoutView.as_view(next_page="/"), name="django-auth-logout"),
     path("oidc/", include('mozilla_django_oidc.urls')),
+
+    path("i18n.js", JavaScriptCatalog.as_view(), name="javascript-catalog"),
```

Comment:
I would prefer if this would be called something with "catalog" in it, so that it is consistent with the other names. Is there anything that?